sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/alchemy.yaml

```
@@ -34,5 +34,4 @@ provider:
           # - https://eth-ropsten.alchemyapi.io/v2/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
           # - https://eth-goerli.alchemyapi.io/v2/00000000000000000000000000000000
           patternNot: 
-            "(?i)\
-            (\\w)\\1{7,}"
\ No newline at end of file
+            - "(?i)(\\w)\\1{7,}"
```

```suggestion
            # https://eth-ropsten.alchemyapi.io/v2/XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX and equivalents
            - "(?i)(\\w)\\1{7,}"
```

Can you remove the comments between post and patternNot also please ?

---

@loris-s-sonarsource : Should I remove the comments or add another one? Your suggestion sounds different than the text you wrote.

---

I mean remove the comments before patternNot, but still keep the one in the suggestion 👍